ipn/localapi/localapi.go

```diff
@@ -1292,12 +1294,53 @@ func (h *Handler) serveFilePut(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "bogus peer URL", http.StatusInternalServerError)
 		return
 	}
-	outReq, err := http.NewRequestWithContext(r.Context(), "PUT", "http://peer/v0/put/"+filenameEscaped, r.Body)
+
+	// Before we PUT a file we check to see if there are any existing partial files and if so,
+	// we resume the upload from where we left off by sending the remaining body instead of
```

Comment:
Nit: "sending the remaining file instead of the full file". Be consistent about "body" vs "file"